Improve Confirmation Modal Design

Yashgabani845 opened the pull request:
This PR include Changes in confirmation model with more appropriate dailog box 
it include chnages in CSS of New project modelm have included minimal changes by adding inline CSS 

fixes : #4044

### **Before**
![image](https://github.com/user-attachments/assets/68c9de10-2a0d-4aa3-9bcb-827d67745b08)

### **After**
![Screenshot (30)](https://github.com/user-attachments/assets/c0433ac9-e11e-47ad-a7cf-89d6e3f7405f)

walterbender commented:
The confirmation deign looks great. But there is a problem in that the way you implement it, by hard-coding it in index.html makes it difficult to include string translation for internationalization. I think you'll want to still create the model-container itself in index.html but set the container's inner html code from JavaScript so we can add the call the _() for string translation.

Yashgabani845 commented:
@walterbender  okay i will make changes and do it from js side

Yashgabani845 commented:
@walterbender done, i have changed the hard code part 
and added required Js code and _() function in toolbar.js and chnaged commit 
can you please check it
